lms/models/application_instance.py

```diff
@@ -38,6 +41,33 @@ class ApplicationInstance(BASE):
         nullable=False,
     )
 
+    # LMS info linked to this consumer_key, description of each one from the spec
+    # https://www.imsglobal.org/specs/ltiv1p1p1/implementation-guide
```

Comment:
I think the comment is out of date because we no longer use the descriptions from the spec. It's probably not needed anyway.

```suggestion
```

If an ORM class has lots of columns and we want to separate them with "heading" comments like this I think it might be better to move them into a separate class that the `ApplicationInstance` class inherits from. That way you get class names and docstrings for documentation and the separate classes naturally structure things. Heading comments tend to get out of sync with the code. You could have something like:

```python
class ApplicationInstance(BASE, _ConsumerKeyColumns, _LMSColumns):
    __tablename__ = "application_instances"
    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
    ...

class _ConsumerKeyColumns:
    """ApplicationInstance columns related to the consumer key."""
    consumer_key = sa.Column(sa.String, unique=True, nullable=False)
    ...

class _LMSColumns:
    """ApplicationInstance columns related to the LMS."""
    ...
```

In this case I think just all in one class is fine though